Sources/XMLCoder/Encoder/XMLEncoder.swift

```diff
@@ -376,6 +376,12 @@ open class XMLEncoder {
         return element.toXMLString(with: header, formatting: outputFormatting)
             .data(using: .utf8, allowLossyConversion: true)!
     }
+
+    // MARK: - TopLevelEncoder
+
+    open func encode<T>(_ value: T) throws -> Data where T: Encodable {
+        try encode(value, withRootKey: nil, rootAttributes: nil, header: nil)
```

Comment:
```suggestion
        return try encode(value, withRootKey: nil, rootAttributes: nil, header: nil)
```

Reply:
I fixed in 84c33e77a57cd008c86321476f5e5a506d80fd76.